packages/rocketchat-version-check/server/functions/checkVersionUpdate.js

```diff
@@ -60,4 +61,39 @@ export default () => {
 			});
 		});
 	}
+
+	if (alerts && alerts.length) {
+		RocketChat.models.Roles.findUsersInRole('admin').forEach((adminUser) => {
+			try {
+				Meteor.runAsUser(adminUser._id, () => Meteor.call('createDirectMessage', 'rocket.cat'));
+
+				alerts.forEach((alert) => {
+					if (RocketChat.models.Users.bannerExistsById(adminUser._id, `alert-${ alert.id }`)) {
+						return;
+					}
+
+					const msg = {
+						msg: `*${ TAPi18n.__(alert.title, adminUser.language) }*\n${ TAPi18n.__(alert.text, ...(alert.textArguments || []), adminUser.language) }\n${ alert.infoUrl }`,
```

Comment:
Should we prefix this some how so they know where this alert is coming from?

Reply:
@geekgonecrazy for example?